Kudu.Services.Web/App_Start/NinjectServices.cs

```diff
@@ -476,6 +476,7 @@ public static void RegisterRoutes(IKernel kernel, RouteCollection routes)
             // Zip push deployment
             routes.MapHttpRouteDual("zip-push-deploy", "zipdeploy", new { controller = "PushDeployment", action = "ZipPushDeploy" }, new { verb = new HttpMethodConstraint("POST", "PUT") });
             routes.MapHttpRoute("zip-war-deploy", "api/wardeploy", new { controller = "PushDeployment", action = "WarPushDeploy" }, new { verb = new HttpMethodConstraint("POST") });
+            routes.MapHttpRouteDual("validate-zip-deploy", "validatezipdeploy", new { controller = "PushDeployment", action = "ValidateZipDeploy" }, new { verb = new HttpMethodConstraint("GET") });
```

Comment:
This should be POST as we are performing validation.

Reply:
Also make sure we can call this from ARM. the route should be api/zipdeploy/validate.  Also no need to Map..Dual - that is for old api.;   New api should be just MapHttpRoute.

Reply:
Thank you for reviewing. 
1. I have tested the API with and without JSON request. I can't test the ARM template locally, so I'll test it on private stamp after adding extension, like I did last time. I'll definitely test it thoroughly there to see if any other changes will be required in the ARM layer for it to work. 
2. I have made the above changes. Are there any other changes required?